### ISSUE TITLE
Difference between NPM packages?

### ISSUE THREAD

**Issue by runn1ng** [zeromq/zeromq.js#137]
I have found those packages on NPM, all seem to do the same thing and from the same authors

https://npmcompare.com/compare/zeromq,zmq,zmq-prebuilt,zmq-static

It seems zeromq is the newest and zmq, zmq-static and zmq-prebuilt are older versions of that library. Am I correct?

**Comment by rgbkrk** [comment]
This repository used to be named `nteract/zmq-prebuilt`, starting as a community run fork of zmq that fixed up the build process and automated prebuilds. In the process of setting up a way to do statically compiled binaries of zeromq for node, we created `zmq-static`. We eventually were able to do that within `zmq-prebuilt` as well as keep the release artifacts + node_modules installation very small. Once we were shipping for awhile and suggesting people use it for electron + node.js, we moved the repository over to the zeromq org and it became official.
